.github/workflows/build-extensions.yml

```diff
@@ -185,8 +185,11 @@ jobs:
           apt install -y libelf-dev zlib1g-dev pkg-config
           apt install -y clang llvm
           apt install -y cargo
+          apt install -y nasm
           # Running tests of wasm_bpf requires proper ebpf running environment
           mount -t debugfs none /sys/kernel/debug
+          bash utils/ffmpeg/install-ffmpeg-v6.0.sh
+          PKG_CONFIG_PATH=$(pwd)/WasmEdge/FFmpeg-n6.0/output/lib/pkgconfig
```

Comment:
The current folder is `WasmEdge`. Instead, you should use `export PKG_CONFIG_PATH=$(pwd)/FFmpeg-n6.0/output/lib/pkgconfig`.
This line should move to line 195 before you run the cmake command. GitHub action will not keep any environment variables between jobs.